tools/add_triage_signature.py

```diff
@@ -532,13 +532,51 @@ def _update_ticket(self, url, issue_key, should_update=False):
                 self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class ConsoleTimeoutSignature(Signature):
+    '''
```

Comment:
replace with triple double-quotes (" x 3)

Reply:
done. 
both are similar: https://www.digitalocean.com/community/tutorials/how-to-format-text-in-python-3#multiple-lines
though for consistency, it's indeed recommended per PEP 257: https://www.python.org/dev/peps/pep-0257/